core/arch/arm/plat-stm/tz_a9init.S

```diff
@@ -238,141 +227,25 @@ _early_a9_r3p0:
 	/*
 	 * GIC configuration
 	 *
-	 * Register ICDISR0 = 0xFFFFFFFF
-	 * - All local interrupts are NonSecure.
+	 * per-CPU interrupts config are in accordance with GIC driver:
+	 *
+	 * ID0-ID7(SGI)   for Non-secure interrupts
+	 * ID8-ID15(SGI)  for Secure interrupts.
+	 * All PPI config as Non-secure interrupts.
 	 *
+	 * Register ICDISR0 = 0xFFFF00FF
 	 * Register ICCPMR = 0xFFFFFFFF
 	 */
 
 	ldr  r0, =GIC_DIST_BASE
-	mov  r1, #0xFFFFFFFF
+	mov  r1, #0xFFFF00FF
```

Comment:
restore the "all-non-secure" interruptions.
plat-stm will drop these and rely on generic gic driver and config.